python/ray/_private/usage/usage_lib.py

```diff
@@ -356,6 +359,12 @@ def _generate_cluster_metadata():
                 "session_start_timestamp_ms": int(time.time() * 1000),
             }
         )
+        if sys.platform == "linux":
+            # Record glibc version
+            (lib, ver) = platform.libc_ver()
+            if lib != "glibc":
+                ver = "na"
```

Comment:
nit: empty string?

Reply:
Any reason for that? I'd prefer `na` if not fetched successfully.

If it's mac or windows, it'll be None.